public/index.html

```diff
@@ -10,10 +10,10 @@
     <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/jointjs/3.2.0/joint.css" />
     <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width"/>
     <!-- Runtime environment variables -->
-    <script src="./config.js"></script>
+    <script src="/config.js"></script>
```

Comment:
not sure if this is good idea ! This assumes that everything is in root, but what if we deploy it to e.g.
https://kbss.felk.cvut.cz/19msmt-demo/fta-fmea